api/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>addresshierarchy</artifactId>
-		<version>2.22.0-SNAPSHOT</version>
+		<version>3.0.0-SNAPSHOT</version>
```

Comment:
If we still support Java 8, is there a reason why we do this major version bump?

Reply:
We are still updating the required platform version to 2.7.0, So don't we need the major version bump?

Reply:
Have we made any change that will fail on a lower platform version like 2.5?

Reply:
We can't compile with platform 2.5.0 with the current changes.

Reply:
Is that because we are using something that was introduced in 2.7?

Reply:
Like which exact one?

Reply:
Platform 2.7.0: `ModuleFactory.getStartedModulesMap().put("exti18n", new Module("", "exti18n", "", "", "", "1.0.0", "") );`

Platform 2.5.0: `ModuleFactory.getStartedModulesMap().put("exti18n", new Module("", "exti18n", "", "", "", "1.0.0") );`